### PR TITLE
Move cache of recently used agents to distribution map

### PR DESCRIPTION
**Pull request by donggyu04**
Move cache of recently used agents to distribution map.

### CODE REVIEW

**Review by songeunwoo — APPROVED** [review]

File: ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java [hunk 1]
```diff
@@ -149,6 +149,7 @@ public CacheConfigHolder cacheConfigMap() {
 		cm.addDistMap(DIST_MAP_NAME_SAMPLING, 15);
 		cm.addDistMap(DIST_MAP_NAME_MONITORING, 15);
 		cm.addDistMap(DIST_MAP_NAME_AGENT, 10);
+		cm.addDistMap(CACHE_RECENTLY_USED_AGENTS, 1 * DAY);
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java:152]: Please rename CACHE_RECENTLY_USED_AGENTS to DIST_ MAP_RECENTLY_USED_AGENTS

**Comment by junoyoon** [comment]
Please describe the problem when you fix the bugs.

**Comment by donggyu04** [comment]
@junoyoon 

We are using hazelcast dist map to store the data of recently used agents.
but in configuration, it's set to a local cache.

Simply the expire time that I set is not being applied. (T_T)

File: ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java (hunk 1 above)
Comment (donggyu04) [reply ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java]: Fixed

File: ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java [hunk 2]
```diff
@@ -149,6 +149,7 @@ public CacheConfigHolder cacheConfigMap() {
 		cm.addDistMap(DIST_MAP_NAME_SAMPLING, 15);
 		cm.addDistMap(DIST_MAP_NAME_MONITORING, 15);
 		cm.addDistMap(DIST_MAP_NAME_AGENT, 10);
+		cm.addDistMap(DIST_MAP_CACHE_RECENTLY_USED_AGENTS, 1 * DAY);
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java:152]: Could u rename all const here? 
I think it's not consistent b/w local and remote cache...

For example, I think the following const is better.

* DIST_CACHE_SAMPLING
* DIST_CACHE_RECENTLY_USED_AGENTS
* LOCAL_CACHE_USERS

Please fix them by seperate commit.

**Review by songeunwoo — APPROVED** [review]

**Pull request merged**